SU-SWS/jumpstart_ui
# D8CORE-3042 Adjusted css for youtube videos in paragraph types

# READY FOR REVIEW

# Summary
- Fixed the iframe containers with videos in cards with multiple per row.

# Need Review By (Date)
- 11/25

# Urgency
- high

# Steps to Test
1. checkout this branch and clear browser cache
1. create a basic page
1. put 3 rows with 1, 2, & 3 items. Each row should contain a card/media w/ caption
1. populate the items with a youtube video
1. save and ensure the youtube videos don't look wrong.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
